src/Sylius/Bundle/ResourceBundle/DependencyInjection/Compiler/ResolveDynamicRelationsPass.php

```diff
@@ -100,4 +100,14 @@ private function getClass(ContainerBuilder $container, $key)
             sprintf('The class %s does not exist.', $key)
         );
     }
+
+    /**
+     * @param $resourceConfig
+     *
+     * @return bool
+     */
+    public function shouldBeResolved($resourceConfig)
+    {
+        return SyliusResourceBundle::DRIVER_DOCTRINE_ORM === $resourceConfig['driver'] && isset($resourceConfig['classes']['interface']);
```

Comment:
`Resolve**Dynamic**RelationsPass`, I guess it miss something to have something dynamic ;)